.github/workflows/deploy.yml

```diff
@@ -39,4 +44,4 @@ jobs:
       run: npm run build
 
     - name: Deploy
-      run: spaship --env="${{ secrets.SPASHIP_ENVIRONMENT }}"
+      run: spaship --env="${{ secrets.SPASHIP_ENVIRONMENT" }}
```

Comment:
The `"` seems off - isn't it supposed to be at the end?
This github action linter complains: https://rhysd.github.io/actionlint/

Reply:
Yeah, I fixed that in another area I'm working on. Sorry, I'll be closing this MR.